extensions/Prefixes.md

```diff
@@ -48,7 +48,7 @@ Once a prefix is approved, it should be added to the [glTF Validator's list of k
 | `MAXAR` | Maxar Technologies | https://www.maxar.com/ | [#1869](https://github.com/KhronosGroup/glTF/issues/1869) |
 | `MESHOPT` | meshoptimizer | https://meshoptimizer.org/ | [#1634](https://github.com/KhronosGroup/glTF/issues/1634) |
 | `MOZ` | Mozilla Corporation | https://www.mozilla.org/ | [#1349](https://github.com/KhronosGroup/glTF/issues/1349) |
-| `MPEG` | The Moving Picture Experts Group | https://mpeg.chiariglione.org/standards/mpeg-i | [#1754](https://github.com/KhronosGroup/glTF/issues/1754) |
+| `MPEG` | The Moving Picture Experts Group | http://mpeg.org | [#1754](https://github.com/KhronosGroup/glTF/issues/1754) |
```

Comment:
Based on the web-server headers, the link should be https://www.mpeg.org/.

Reply:
implemented.